jgrapht-core/src/test/java/org/jgrapht/alg/matching/blossom/v5/BlossomVInitializerTest.java

```diff
@@ -360,12 +358,12 @@ public void testFractionalInitialization9() {
         for (int[] edge : edges) {
             Graphs.addEdgeWithVertices(graph, edge[0], edge[1], edge[2]);
         }
-        KolmogorovMinimumWeightPerfectMatching<Integer, DefaultWeightedEdge> perfectMatching = new KolmogorovMinimumWeightPerfectMatching<>(graph, fractionalOptions);
+        KolmogorovWeightedPerfectMatching<Integer, DefaultWeightedEdge> perfectMatching = new KolmogorovWeightedPerfectMatching<>(graph, fractionalOptions);
         MatchingAlgorithm.Matching<Integer, DefaultWeightedEdge> matching = perfectMatching.getMatching();
 
         assertEquals(17, matching.getWeight(), EPS);
         assertTrue(perfectMatching.testOptimality());
-        checkMatchingAndDualSolution(matching, perfectMatching.getDualSolution());
+        checkMatchingAndDualSolution(matching, perfectMatching.getDualSolution(), false);
 
     }
```

Comment:
Do we need tests for the max weight perfect matching case here as well?

Reply:
I think, no, because initialization doesn't depend on the objective sense of the algorithm or whether the matching should be perfect or not: it is always done as part of the min. weight perfect matching algorithm.